build-tools-internal/version.properties

```diff
@@ -19,10 +19,8 @@ commons_lang3     = 3.9
 google_oauth_client = 1.34.1
 
 antlr4            = 4.13.1
-# when updating this version, you need to ensure compatibility with:
-#  - distribution/tools/plugin-cli
```

Comment:
plugin-cli uses the fips variant: https://github.com/elastic/elasticsearch/blob/main/distribution/tools/plugin-cli/build.gradle#L30